Resources/Prototypes/Corvax/Voice/speech_emotes.yml

```
@@ -54,13 +54,17 @@
   name: chat-emote-name-purr
   category: Vocal
   available: false
-  icon: Corvax/Interface/Emotes/purr.png
+  icon: _WL/Interface/Emotes/purr.png # WL-changes
```

```suggestion
  icon: _WL/Interface/Emotes/purr.png # WL-Changes: Purr sound change
```
Стоит везде в WL-Changes комментариях указывать тему